documentation/configuration/configfile.md

```diff
@@ -151,6 +151,9 @@ Here is a sample configuration file:
 #
 flyway.locations=filesystem:sql
 
+# Whether to fail if a location specified in the flyway.locations option doesn't exist (default: false)
+# flyway.failOnMissingLocations=
+
```

Comment:
We don't need to/shouldn't manually update the conf in this file - the release notes automater will do it for us